src/ui/RealmBrowser/Content/CreateObjectDialog/types/DefaultControl.tsx

```diff
@@ -20,13 +20,20 @@ import React from 'react';
 import { Alert } from 'reactstrap';
 import Realm from 'realm';
 
+export type AlertColorType = 'info' | 'warning' | 'danger';
+
 export interface IDefaultControlProps {
   property: Realm.ObjectSchemaProperty;
   message?: string;
+  type?: AlertColorType;
```

Comment:
I suggest using a different word here: To me, `type` refers the the type of the property in a lot of other places.

```suggestion
  color?: AlertColorType;
```

Reply:
I don't really see `'info' | 'warning' | 'danger'` as colors..?

Reply:
They are names of colors defined as SCSS variables (see https://github.com/realm/realm-studio/blob/master/styles/variables/_colors.scss and https://reactstrap.github.io/components/alerts/).

Reply:
Ok, I'll yield. The naming is set by bootstrap..